src/index.ts

```diff
@@ -1104,6 +1136,23 @@ function handleMeetingStatus(message: ZoomOSCMessage) {
     }
 }
 
+function handlePongReply(message: ZoomOSCMessage) {
+
+    ZoomOSCInfo = {
+        pingArg: message.userName,
+        zoomOSCversion: String(message.galIndex),
+        subscribeMode: Number(message.zoomID),
+        galTrackMode: Number(message.params[0]),
+        callStatus: Number(message.params[1]),
+        numTargets: Number(message.params[2]),
+        numUsers: Number(message.params[3]),
+        isPro: Boolean(message.params[4])
```

Comment:
I see what you mean. You'd need a whole new message type to handle that more cleanly. Another approach would be to skip the parserZoomOSCMessage() function and just parse it in here in a more generic way.